gateway-ha/src/test/java/io/trino/gateway/ha/router/TestQueryCountBasedRouter.java

```diff
@@ -130,11 +130,10 @@ static ClusterStats getClusterWithNoUserQueueAndMinQueueCount()
         cluster.routingGroup("adhoc");
         cluster.runningQueryCount(5);
         cluster.queuedQueryCount(LEAST_QUEUED_COUNT);
-        cluster.userQueuedCount(new HashMap<String, Integer>());
```

Comment:
I assume this is triggers the scenario in the test now .. do we need another non-null test?

Reply:
That's correct. This is sufficient, I verified that it does validate the scenario which is being fixed.